src/Network/Ethereum/Web3/Solidity/Generic.purs

```diff
@@ -201,39 +203,51 @@ instance argsToRowListProxyBase :: ArgsToRowListProxy (Argument (Tagged (SProxy
 instance argsToRowListProxyInductive :: ArgsToRowListProxy as l => ArgsToRowListProxy (Product (Argument (Tagged (SProxy s) a)) as) (Cons s a l) where
   argsToRowListProxy _ = RLProxy
 
-class ToRecordFields args fields (rowList :: RowList) | args -> rowList, rowList -> args fields where
+class ListToRow rowList fields <= RecordFieldsIso args fields (rowList :: RowList) | args -> rowList, rowList -> args fields where
```

Comment:
Also `ListToRow rowList fields` implies  `rowList -> fields, fields -> rowList` , So I think we would be fine with just:
`args -> rowList, rowList -> args` or
`args -> fields, rowList -> fields`